test/longest_string_test.js

```diff
@@ -0,0 +1,14 @@
+const assert = require('chai').assert;
+const longestString = require('../lib/longest_string');
+
+describe('Longest String', function() {
+  it('returns the longest string in array', function() {
+    var array = ['horses', 'cats', 'pineapples'];
```

Comment:
you're using `const` and `var` here - i encourage you to choose one and stick with it 😄